src/oas-parsing/operation-example/operation-example.factory.ts

```diff
@@ -1,23 +1,49 @@
+import ExampleGroup from '../example-group/example-group';
 import OASOperation from '../operation/oas-operation';
 import OperationExample from './operation-example';
 
 export default class OperationExampleFactory {
   public static buildFromOperations(
     operations: OASOperation[],
+  ): OperationExample[] {
+    let operationExamples: OperationExample[] = [];
+
+    operations.forEach((operation) => {
+      operationExamples = [
+        ...operationExamples,
+        ...this.buildFromOperation(operation),
+      ];
+    });
+
+    return operationExamples;
```

Comment:
```suggestion
    return operations.flatMap((operation) => this.buildFromOperation(operation));
```

I think using [flatMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap) would help simplify this code (and below)